### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by owenconnorz** [z-huang/music#126] [General]
Hi would u be interested in a discord server I'm happy to make it and also pay for the server and have feature request bug reports and ect 😁

**Comment by z-huang** [comment]
Thanks for your contribution. However, as said in [comment](https://github.com/z-huang/music/issues/86#issuecomment-1151218306), I want things to stay in GitHub. Also, I can't be active on this project all the time.

**Reply by owenconnorz** [reply]
No worries dude if u need anything let me know anytime 🤟
